android/src/main/java/com/appcuesreactnative/AppcuesFrameViewManager.kt

```diff
@@ -0,0 +1,69 @@
+package com.appcuesreactnative
+
+import android.content.Context
+import android.view.View
+import android.widget.FrameLayout
+import androidx.core.view.children
+import com.appcues.AppcuesFrameView
+import com.facebook.react.uimanager.ThemedReactContext
+import com.facebook.react.uimanager.UIManagerModule
+import com.facebook.react.uimanager.ViewGroupManager
+import com.facebook.react.uimanager.annotations.ReactProp
+
+
+internal class AppcuesFrameViewManager: ViewGroupManager<AppcuesWrapperView>() {
+
+    override fun getName() = REACT_CLASS
+
+    companion object {
+        const val REACT_CLASS = "AppcuesFrameView"
+    }
+
+    override fun createViewInstance(context: ThemedReactContext): AppcuesWrapperView {
+        return AppcuesWrapperView(context)
+    }
+
+    @ReactProp(name = "frameID")
+    fun setFrameId(view: AppcuesWrapperView, frameId: String) {
+        AppcuesReactNativeModule.implementation?.registerEmbed(frameId, view.contentView)
+    }
+}
+
+internal class AppcuesWrapperView(context: Context) : FrameLayout(context) {
+    val contentView: AppcuesFrameView = AppcuesFrameView(context)
```

Comment:
I messed this up, it's not getting added to the parent view now - will patch this up.